ios/Exponent/Versioned/Core/UniversalModules/Permissions/EXScopedPermissions.m

```diff
@@ -120,7 +120,7 @@ - (NSDictionary *)getScopedPermissionForType:(NSString *)permissionType withGlob
       && [self shouldVerifyScopedPermission:permissionType]
       && [EXPermissions statusForPermission:permission] == UMPermissionStatusGranted) {
     permission[@"status"] = [self getScopedPermissionStatus:permissionType];
-    permission[@"granted"] = @(NO);
+    permission[@"granted"] = [permission[@"status"] isEqual:@"granted"] ? @YES : @NO;
```

Comment:
Oops, this issue seems to be critical and the fix should probably be released to the store 🤔

Reply:
I'm not sure, cause checking the "status" field gives you the correct value. Moreover, it was since SKD 36.

Reply:
And also it only occurs in the client.